docs/source/apis/aws_marketplace.rst

```diff
@@ -0,0 +1,245 @@
+
+.. image:: imgs/AWS/VPC_Step6.png
+  :align: center
+  :alt: connect public subnet to route table
+
+.. image:: imgs/AWS/VPC_Step7.png
+  :align: center
+  :alt: VPC Resource Maps
+
+7. **Inspect VPC Resource Map**:
+
+   You can check the configurations from the Resource Maps on the VPC Details dashboard.
+
+Part 2: Deploying Unstructured API from AWS Marketplace
+-------------------------------------------------------
+
+8. **Visit the Unstructured API page on AWS Marketplace**
+
+   - Link: `Unstructured API Marketplace <http://aws.amazon.com/marketplace/pp/prodview-fuvslrofyuato>`_.
+   - Click ``Continue to subscribe``
+   - Review Terms and Conditions
+   - Click ``Continue to Configuration``
+
+.. image:: imgs/AWS/Marketplace_Step8.png
+  :align: center
+  :alt: Unstructured API on AWS Marketplace
+
+9. **Configure the CloudFormation**
+
+   - Select ``CloudFormation Template`` from the Fulfillment option dropdown menu.
+   - Use the default ``Unstructured API`` template and software version.
+   - Select the ``Region``
+
+     - *Note: select the same region where you set up the VPC in Part 1.*
+   - Click ``Continue to Launch`` button.
+   - Select ``Launch CloudFormation`` from Choose Action dropdown menu.
+   - Click ``Launch`` button.
+
+
+.. image:: imgs/AWS/Marketplace_Step9.png
+  :align: center
+  :alt: CloudFormation Configuration
+
+
+10. **Create Stack on CloudFormation**
+
+    The Launch button will redirect to ``Create stack`` workflow in the CloudFormation.
+
+    **Step 1: Create stack**
+
+    - Select the ``Template is ready``
+    - Use the default template source from ``Amazon S3 URL``
+    - Click ``Next`` button.
+
+    .. image:: imgs/AWS/Marketplace_Step10a.png
+        :align: center
+        :alt: Create Stack
+
+
+    **Step 2: Specify stack details**
+
+    - Provide ``stack name``
+    - In the **Parameters** section, provide the ``KeyName``, ``Subnets``, and ``VPC`` from Part 1 above.
+    - Specify, ``LoadBalancerScheme`` to **internet-facing** and ``SSHLocation`` to  **0.0.0.0/0**
+    - Click ``Next`` button.
+
+    .. image:: imgs/AWS/Marketplace_Step10b.png
+        :align: center
+        :alt: Specify stack details
+
+    **Step 3: Configure stack options**
+
+    - Specify the stack options or use default values.
+    - Click ``Next`` button.
+
+    .. image:: imgs/AWS/Marketplace_Step10c.png
+        :align: center
+        :alt: Specify stack options
+
+    **Step 4: Review**
+
+    - Review the Stack settings.
+    - Click ``Submit`` button.
+
+    .. image:: imgs/AWS/Marketplace_Step10d.png
+        :align: center
+        :alt: Review stack
+
+
+11. **Get the Unstructured API Endpoint**
+
+    - Check the status of the CloudFormation stack.
+
+      - A successful deployment will show ``CREATE_COMPLETE`` status.
+    - Click ``Resources`` tab and find ``ApplicationLoadBalancer``.
+    - Copy the ``DNS Name`` from the Load Balancer dashboard.
```

Comment:
I had to also click the ApplicationLoadBalancer to launch a page that had the details with the DNS Name. ... if this is the case, please specify that explicitly